pydm/default_stylesheet.qss

```diff
@@ -0,0 +1,201 @@
+/*
+GREEN_ALARM = "#00EB00"		
+YELLOW_ALARM = "#EBEB00"		
+RED_ALARM = "#FF0000"		
+MAGENTA_ALARM = "#EB00EB"		
+WHITE_ALARM = "#EBEBEB"
+*/
+/*
+ALARM_NONE = 0
+ALARM_MINOR = 1
+ALARM_MAJOR = 2
+ALARM_INVALID = 3
+ALARM_DISCONNECTED = 4
+*/
+
+/* =================== Alarm-sensitive borders ===================== */
+
+/* Alarm severity == ALARM_NONE */
+*[alarmSensitiveBorder="true"][alarmSeverity="0"] {
+    border: 2px solid #00EB00;
+}
+
+/* Alarm severity == ALARM_MINOR */
+*[alarmSensitiveBorder="true"][alarmSeverity="1"] {
+    border: 2px solid #EBEB00;
+}
+
+/* Alarm severity == ALARM_MAJOR */
+*[alarmSensitiveBorder="true"][alarmSeverity="2"] {
+    border: 2px solid #FF0000;
+}
+
+/* Alarm severity == ALARM_INVALID */
+*[alarmSensitiveBorder="true"][alarmSeverity="3"] {
+    border: 2px solid #EB00EB;
+}
+
+/* Alarm severity == ALARM_DISCONNECTED */
+*[alarmSensitiveBorder="true"][alarmSeverity="4"] {
+    border: 2px dashed #FFFFFF;
+}
+
+/* We have to call out the custom pen on PyDMDrawingWidget */
+PyDMDrawing[alarmSensitiveBorder="true"] {
+    border: none;
+    qproperty-penWidth: 2px;
+    qproperty-penStyle: SolidLine;
+}
+
+PyDMDrawing[alarmSensitiveBorder="true"][alarmSeverity="0"] {
+    qproperty-penColor: #00EB00;
+}
+
+PyDMDrawing[alarmSensitiveBorder="true"][alarmSeverity="1"] {
+    qproperty-penColor: #EBEB00;
+}
+
+PyDMDrawing[alarmSensitiveBorder="true"][alarmSeverity="2"] {
+    qproperty-penColor: #FF0000;
+}
+
+PyDMDrawing[alarmSensitiveBorder="true"][alarmSeverity="3"] {
+    qproperty-penColor: #EB00EB;
+}
+
+PyDMDrawing[alarmSensitiveBorder="true"][alarmSeverity="4"] {
+    qproperty-penColor: #FFFFFF;
+    qproperty-penStyle: DashLine;
+}
+
+/* =============== Alarm-sensitive content ================= */
+PyDMCheckbox[alarmSensitiveContent="true"][alarmSeverity="0"],
+PyDMSpinbox[alarmSensitiveContent="true"][alarmSeverity="0"],
+PyDMEnumComboBox[alarmSensitiveContent="true"][alarmSeverity="0"],
+PyDMPushButton[alarmSensitiveContent="true"][alarmSeverity="0"] {
+	background-color: #00EB00;
+}
+
+PyDMCheckbox[alarmSensitiveContent="true"][alarmSeverity="1"],
+PyDMSpinbox[alarmSensitiveContent="true"][alarmSeverity="1"],
+PyDMEnumComboBox[alarmSensitiveContent="true"][alarmSeverity="1"],
+PyDMPushButton[alarmSensitiveContent="true"][alarmSeverity="1"] {
+	background-color: #EBEB00;
+}
+
+PyDMCheckbox[alarmSensitiveContent="true"][alarmSeverity="2"],
+PyDMSpinbox[alarmSensitiveContent="true"][alarmSeverity="2"],
+PyDMEnumComboBox[alarmSensitiveContent="true"][alarmSeverity="2"],
+PyDMPushButton[alarmSensitiveContent="true"][alarmSeverity="2"] {
+	background-color: #FF0000;
+}
+
+PyDMCheckbox[alarmSensitiveContent="true"][alarmSeverity="3"],
+PyDMSpinbox[alarmSensitiveContent="true"][alarmSeverity="3"],
+PyDMEnumComboBox[alarmSensitiveContent="true"][alarmSeverity="3"],
+PyDMPushButton[alarmSensitiveContent="true"][alarmSeverity="3"] {
```

Comment:
I thought we can do this:

PyDMWidget, PyDMWritableWidget, PyDMLabel, PyDMLineEdit, PyDMSlider, PyDMCheckbox[alarmSeverity="4"][alarmSensitiveBorder="true"][alarmSensitiveContent="true"]{color: #EBEBEB; border: 2px solid #EBEBEB}

Reply:
No, otherwise you are applying the selector only to PyDMCheckbox...
The comma separates the selectors and the `[alarm...]` is part of the selector.